bin/run_controller

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash -e
+
+ROOT=$(realpath $(dirname $0)/..)
+CONTAINER=forch-controller
+INSTDIR=$ROOT/inst
+IMAGE=forch/controller
+PROM_TARGET=9302
+GAUGE_PROM_TARGET=9303
+FAUCET_SOCK=faucet_event.sock
+FAUCET_CONFIG=faucet.yaml
+SYSTEM_CONF=system.conf
+LOCAL_CONFIG=local/$SYSTEM_CONF
+
+ext_ofpt=6653
+gauge_ext_ofpt=6654
+prom_pt=9302
+gauge_prom_pt=9303
+
+docker_volumes=
+port_opts=
+local=
+faucet_id=
+
+cd $ROOT
+
+if [ "$1" == "local" ]; then
+    docker_volumes+=" -v $ROOT/faucet/faucet:/usr/lib/python3.7/site-packages/faucet"
+    local=local
+    shift
+fi
+
+if [ -n "$1" ]; then
+    CONTAINER=$CONTAINER-$1
+    INSTDIR=$INSTDIR/forch-controller-$1
+    faucet_id=$1
+    shift
+fi
+
+if [ -n "$1" ]; then
```

Comment:
this likely isn't the best way to do this because it ends up as a string of numerical arguments that all look the same... but,  don't need to fix it now -- just realize that it's a messy way to do it. Something that works for n<=3 often doesn't scale for n>3 since it tips over from "simple is easy" to "simple is messy"

Reply:
I will keep this in mind when I do the test_fot script.